### ISSUE TITLE
Declutter view talk page

### ISSUE THREAD

**Issue by poonen** [roed314/seminars#357]
On each "View talk" page:

1) The "Livestream access" box does not deserve to have its own line.  It should be moved to the right of the text "Mon May 4, 07:30-08:30 (starts in 12 hours)".

2) I think we could simplify

Talk content
   * paper
   * slides
   * video

to
(paper | slides | video)
which simplifies further to
(slides | video)
(slides)
or nothing at all, depending on how many items are present.
Or if we really want to keep the words "Talk content", at least put everything on one line:
Talk content: (paper | slides | video)

**Comment by AndrewVSutherland** [comment]
I agree, in fact I don't understand why this was changed, I liked the ( paper | slides | video ) presentation we had before, and I don't think the phrase "Talk content" is necessary.

**Comment by poonen** [comment]
I'd forgotten that that's how we had it before; maybe I was advocating the other way earlier!  I guess the argument in favor of the text "Talk content" is to give the page some structure instead of having it be a long list of lines, each with its own thing.  But in this case, I think it is already set off from the other items in the current layout with the <hr>s and the dark topics.

**Comment by AndrewVSutherland** [comment]
OK, I'll make this change and push it to master so you can give feedback.

**Comment by AndrewVSutherland** [comment]
@poonen These changes should be on master in < 60 seconds.  Let me know what you think, and also whether we want to keep Abstract on the view talk page but remove it from the talk knowl.

**Comment by poonen** [comment]
I would definitely remove "Abstract" from the talk knowl.  But maybe we can keep it on the view talk page.

On the view talk page, there needs to be more space between the time and the "Lecture held in".  Or else maybe end the time with a boldface period.

Maybe try centering the 
   (paper | slides | video )
instead of putting it on the left, on both the knowl and the view talk?

**Comment by AndrewVSutherland** [comment]
I changed the "Abstract" line to just a bold "Abstract:" preceding the abstract on both the talk knowl and talk pages, which takes up no more space but I think looks a bit better than omitting it entirely.

On the view talk page, when there is a room specified the room and livestream link now appear below the time, rather than immediately to the right, which is also how they appear in the talk knowl.  Putting the time, room, and live link all on one line can easily make the line to long to read on a tablet or phone (especially if the room is something like "Famous-donor-name memorial lecture hall").  But if there is no room, the live link goes next to the time on the View Talk page.

I tried centering the content links but I think they look better left aligned so I left this as is.

These changes are now visible on master, let me know what you think.
